content/questions/foundations/question-116.md

```diff
@@ -0,0 +1,14 @@
+---
+question: "What are some actions you can do in regards to Project Templates in your organization? (Choose three.)"
+archetype: "questions"
+title: "Question 116"
+draft: true
+---
+
+
+> https://docs.github.com/en/issues/planning-and-tracking-with-projects/managing-your-project/managing-project-templates-in-your-organization
+- [ ] With write permission, set an existing project as a template
+- [x] Configure recommended templates to your organization's members
+- [x] Create a new template to be used as a base for new projects
+- [x] With admin or write permissions, copy an existing project as a template
+- [ ] Publish your templates in GitHub Marketplace for anyone to use
```

Comment:
pre-commit checks are [failing](https://github.com/FidelusAleksander/ghcertified/actions/runs/9211690523/job/25341633758?pr=229) due to missing newline at the end of file

Reply:
I see... gonna fix that